core/templates/dev/head/components/answer_group_editor_directive.html

```diff
@@ -57,7 +57,8 @@
               <[rule | parameterizeRuleDescription:getCurrentInteractionId():answerChoices]>...
             </span>
             <span ng-if="getCurrentInteractionId() === 'MultipleChoiceInput'">
-              <span class="oppia-multiple-choice-rule" angular-html-bind="rule | parameterizeRuleDescription:getCurrentInteractionId():answerChoices"></span>...
+              <span><angular-html-bind class="oppia-multiple-choice-rule" html-data="rule | parameterizeRuleDescription:getCurrentInteractionId():answerChoices"></angular-html-bind>...
```

Comment:
Try using `ng-bind-html` instead of `angular-html-bind` here and elsewhere. See [this guide](https://docs.angularjs.org/api/ng/directive/ngBindHtml).

Reply:
Just wondering, what's wrong with angular-html-bind? I think we use it for this purpose throughout the codebase, so if there's a need to change to ng-bind-html it will affect other fields as well.

Reply:
Ah, in that case, it won't be necessary. I was concerned since it differed from the Angular docs and wondered if that could have caused the errors that Apurv was having.